compiler/modules/front/resolveTypes.bal

```diff
@@ -254,8 +254,10 @@ function resolveTypeDesc(ModuleSymbols mod, s:ModuleLevelDefn modDefn, int depth
     if td is s:ErrorTypeDesc {
         return t:errorDetail(check resolveTypeDesc(mod, modDefn, depth, td.detail));
     }
-    if td is s:UnaryTypeDesc {
-        t:SemType ty = check resolveTypeDesc(mod, modDefn, depth, td.td);
+    // XXX remove td2 and panic when migrated to jballerina beta4
```

Comment:
The comment is wrong: we will still need the `panic` even if beta4 fixes its bug.